application/app/Models/User.php

```diff
@@ -64,4 +70,84 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    /**
+     * @param $query
+     * @param $name
+     * @return mixed
+     */
+    public function scopeFuzzyName($query, $name)
```

Comment:
`use Illuminate\Database\Eloquent\Builder;`

```suggestion
    public function scopeFuzzyName(Builder $query, ?string $name)
```